main/notify.py

```diff
@@ -1,15 +1,14 @@
-"""Function to send notifications about project status."""
+"""Function to send notifications to project lead."""
 
 from django.core.mail import send_mail
 
 
-def email_lead_project_status(email: str, project_name: str, message: str) -> None:
-    """Send email notification to the project lead about the project status."""
-    subject = f"[Project Status Update] {project_name}"
+def email_lead(subject: str, email: str, message: str) -> None:
```

Comment:
Maybe this should just be `email_user`, because it is not just for project leads anymore.